pom.xml

```diff
@@ -536,6 +542,27 @@ Import-Package: \\
       </plugins>
     </pluginManagement>
 
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>enforce-java</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
+                  <version>[1.8.0-40,1.9)</version>
```

Comment:
doesn't this also mean, that java 1.9 is allowed?

Reply:
Follow the documentation link above and click on the "range" (for version ranges) link there:
https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

Reply:
Sorry, I've read `[1.8.0-40,1.9]` here (`[` instead of `)`). Sorry! :)